### PR TITLE
📦  Package the TrustScore and make it compatible with python 3

### PR DESCRIPTION
**Pull request by courentin**

### CODE REVIEW

**Comment by courentin** [comment]
@googlebot I signed it!

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+with open("README.rst", "r", encoding="utf-8") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="trustscore", # Replace with your own username
+    version="0.0.1",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/lalalilo/TrustScore",
```
Comment (hhjiang) [inline setup.py]: should the url point to the original google repo?

File: trustscore/__init__.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+try:
+    # Python 2 forward compatibility
```
Comment (hhjiang) [inline trustscore/__init__.py]: perhaps we don't need this if we're calling range instead of xrange

**Review by hhjiang — COMMENTED** [review]
Thanks for doing this! Looks good overall.
Would you be able to add a simple unit test just to make sure the library runs?

**Comment by courentin** [comment]
Thanks @hhjiang for you review, I took your comments into account!

**Comment by courentin** [comment]
Hello, any plans to merge it? cc @hhjiang

**Pull request merged**